fitbenchmarking/fitbenchmarking
# add levmar support for bounds

#### Description of Work

Fixes [#744](https://github.com/fitbenchmarking/fitbenchmarking/issues/744)

Adding in support for bounds for levmar fitting software. Also includes code overlap as described in [#745](https://github.com/fitbenchmarking/fitbenchmarking/issues/745), will address this in a separate PR.

#### Testing Instructions

1. Run Fitbenchmarking with levmar fitting software
2. Run with sasview examples and check that bounds are respected
3. Run with other benchmark problems to check that results are consistent

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
